src/hooks/use-dhis2-stack-parameters.ts

```diff
@@ -0,0 +1,50 @@
+import { useMemo } from 'react'
+import { Stack, StackParameter } from '../types'
+import { useAuthAxios } from './use-auth-axios'
+
+type SecondaryAndPrimaryParameters = {
```

Comment:
Good point. But since we don't have this yet, let's keep it here for now. We can remove it once the backend returns the required information.

FYI: when I worked on this PR I actually tried to work as generically as possible. There are a few things hardcoded on the client, but it should be fairly easy to turn this DHIS2 Core form into something that can produce a form like this for any type of stack.